arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1175,13 +1179,19 @@ void ClusterInfo::loadPlan() {
   {
     std::lock_guard w(_waitPlanLock);
     triggerWaiting(_waitPlan, idx);
+    // TODO Wouldn't it be better to notify only when the plan version changes?
```

Comment:
Follower DBServer look at the Current entry to determine if they have to initalize a synchronizeShard action. So I would say, no.

Reply:
So the above triggers Futures that wait for _any_ update no matter what it caused?

Reply:
I'll remove the comment, I think its fine.

`triggerWaiting` triggers only futures waiting for a raft index not newer than the one we saw here.